### PR TITLE
make docker use all 4 cores

### PR DESCRIPTION
**Pull request by drunkirishcoder**
4 is better than 2.

### CODE REVIEW

File: docker.mk [hunk 1]
```diff
@@ -11,7 +11,7 @@ REGISTRY_IMG_NAME = netbricks
 LINUX_HEADERS = -v /lib/modules:/lib/modules -v /usr/src:/usr/src
 DPDK_VER = 17.08
 BASE_DIR ?= $(or $(shell pwd),~/occam/Netbricks)
-MAX_CORES ?= 1
+MAX_CORES ?= 3
```
Comment (zeeshanlakhani) [inline docker.mk]: maybe remove this the. However, not sure why cpuset-cpus falters here, according to the docs: https://docs.docker.com/engine/reference/run/#cpuset-constraint? @drunkirishcoder

Comment (drunkirishcoder) [reply docker.mk]: failed because travis runner didn't have access to 4 cores.

Comment (zeeshanlakhani) [reply docker.mk]: gotcha. boom.

**Comment by zeeshanlakhani** [comment]
👍

**Pull request merged**